protobuf/syft_proto/types/syft/v1/shape.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package syft_proto.types.syft.v1;
+
+message Shape {
```

Comment:
I remember there's also torch.Size in serde. 
It can be serialized to this type. Or this one renamed to types/torch/v1/size.proto.

Or maybe we shouldn't have torch/syft distinction in protocol at all?

Reply:
Yeah, I think this might be an artifact of my process of figuring out how this was all going to fit together. It probably should be `types/torch/v1/size.proto.`

I do want to maintain some distinction between the types defined in Syft and the types defined in Torch though. This one just fell on the wrong side of the line due to my confusion about what exactly a "generic" tensor was.

Reply:
(I have additional schemas for Torch types coming next, so I'll tweak it in the next PR.)

Reply:
@vvmnnnkv Renamed types/syft/v1/shape.proto to types/torch/v1/size.proto in #23.